style/modules/pressModule.css

```diff
@@ -0,0 +1,37 @@
+.pressModule {
```

Comment:
let's just go with `.press` for the name -- the reason is that since all of our (non-layout) classes are modules, we can leave "module" off and get shorter names.